Good-Bids/goodbids
# [#130] Adding Oauth plugin

# Summary

This adds the Mini Orange plugin to the MU plugins. We can't pull this plugin using Composer because the plugin is a premium version. 

I set up a test oauth for Google and added the shortcode under the login form. 

We are adding this onto staging to finish the set up with:
- GSuite / Google Apps
- Apple
- Facebook / Meta
- LinkedIn
- Twitter / X

## Issues

* #130

## Testing Instructions

1. Go to the Network dashboard and make sure you see the mini orange navigation link.

## Screenshots

| Login | Login and Register |
|--------|--------|
| <img width="1355" alt="Screenshot 2024-02-08 at 3 07 30 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/671254d4-b519-4826-a239-64274f636a53"> | <img width="1059" alt="Screenshot 2024-02-08 at 3 07 45 PM" src="https://github.com/Good-Bids/goodbids/assets/91974372/c79a46a3-d556-42aa-81da-e0832fd931d9"> |
